apps/fluent-tester/src/FluentTester/TestComponents/Test.tsx

```diff
@@ -64,55 +66,57 @@ export const Test = (props: TestProps) => {
       <Stack style={stackStyle}>
         <Text style={styles.description}>{props.description}</Text>
       </Stack>
-      <Stack style={stackStyle}>
-        <View style={styles.statusView}>
-          <Stack>
-            <Text style={[styles.statusHeader]} variant="headerStandard">
-              Platform Status
-            </Text>
-            <Text style={[styles.statusLabel]} variant="bodySemibold">
-              Win32: <Text style={styles.status}>{props.status.win32Status}</Text>
-            </Text>
-            <Text style={[styles.statusLabel]} variant="bodySemibold">
-              UWP: <Text style={styles.status}>{props.status.uwpStatus}</Text>
-            </Text>
-            <Text style={[styles.statusLabel]} variant="bodySemibold">
-              iOS: <Text style={styles.status}>{props.status.iosStatus}</Text>
-            </Text>
-            <Text style={[styles.statusLabel]} variant="bodySemibold">
-              macOS: <Text style={styles.status}>{props.status.macosStatus}</Text>
-            </Text>
-            <Text style={[styles.statusLabel]} variant="bodySemibold">
-              Android: <Text style={styles.status}>{props.status.androidStatus}</Text>
-            </Text>
-          </Stack>
+      {isMobile ? null : (
```

Comment:
Rather than define it null here, could we just do the platform check wherever we render this component and not render it for mobile paltforms?

Reply:
updated to render on non-mobile platforms